frontend/src/pages/SimpleViewer.jsx

```diff
@@ -39,7 +39,7 @@ export default function SimpleViewer(props) {
 
     const questionId = response.id;
     // Upload question data
-    const questionData = JSON.stringify({ query_graph: messageStore.message.query_graph });
+    const questionData = JSON.stringify({ query_graph: messageStore.message.query_graph }, null, 2);
```

Comment:
It doesn't make much sense to me why we'd be putting spaces in any messages sent to robokache, seems like that would just make the files on disk larger. Wouldn't it make more sense to only add spaces when a user wants to download the json for a message?